### DISCUSSION TITLE
Geolib 0.1.8 on PyPi

### DISCUSSION THREAD

**Discussion by ghost** [Deltares/GEOLib#23] [General]
Hi @carloslubbers,

Thanks for releasing Geolib 0.1.8 and merging my suggestion.
Will the new release be uploaded to PyPi? It is now still at 0.1.7.

greeting,

Maarten

**Comment by carloslubbers** [comment]
Hi @Mbweehuizen, yes it will be! The person who is authorized to do this is currently on holiday, we will upload it to PyPi when he returns and make sure we can do it faster next time :)

**Comment by carloslubbers** [comment]
Apologies for the delays @Mbweehuizen , but we have updated the release on PyPi!

**Reply by ghost** [reply]
thanks
